File: library-base/src/main/java/com/my/library_base/debug/DebugApplication.java
package com.my.library_base.debug;

import android.app.Application;
import android.database.DatabaseErrorHandler;
import android.database.sqlite.SQLiteDatabase;

import com.my.library_base.init.ModuleLifecycleConfig;

public class DebugApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        //初始化组件(靠前)
        ModuleLifecycleConfig.getInstance().initModule(this);
    }

    @Override
    public void onTerminate() {
        super.onTerminate();
        ModuleLifecycleConfig.getInstance().destroyModule(this);
    }
}

File: library-base/src/main/java/com/my/library_base/debug/DebugApplication.kt
package com.my.library_base.debug

import android.app.Application
import com.my.library_base.init.ModuleLifecycleConfig

class DebugApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        //初始化组件(靠前)
        ModuleLifecycleConfig.instance.initModule(this)
    }

    override fun onTerminate() {
        super.onTerminate()
        ModuleLifecycleConfig.instance.destroyModule(this)
    }
}